src/components/portal/portal.tsx

```diff
@@ -175,9 +176,7 @@ export class Portal {
     @Watch('visible')
     protected onVisible() {
         if (!this.visible) {
-            this.hideContainer();
-            this.styleContainer();
```

Comment:
There are some inline styles on `<div class="limel-portal--container"` that control the placement (such as `position`, `transform`, `inset`, and `z-index`) in relation to the trigger element. They are being removed immediately when the portal gets hidden (e.g. scrim-clicking the menu-surface). 

Since I need these styles to persist during the "closing" animation (when the `is-closing` class is temporarily applied), I need to ensure that those inline styles remain intact until the animation finishes. Otherwise, the surface (portal-container) will move out of the viewport, as soon as the portal is closed.

So I had to make sure to keep all placement-related styles intact during the closing animation.
And only after the is-hiding animation completes, clear the inline styles.